src/doc/rustc/src/codegen-options/index.md

```diff
@@ -497,27 +497,27 @@ point instructions in software. It takes one of the following values:
 This option controls the emission of "split debuginfo" for debug information
 that `rustc` generates. The default behavior of this option is
 platform-specific, and not all possible values for this option work on all
-platform. Possible values are:
+platforms. Possible values are:
 
 * `off` - This is the default for platforms with ELF binaries and windows-gnu
-  (not Windows MSVC and not macOS). This typically means that dwarf debug
+  (not Windows MSVC and not macOS). This typically means that DWARF debug
   information can be found in the final artifact in sections of the executable.
   This option is not supported on Windows MSVC. On macOS this options prevents
   the final execution of `dsymutil` to generate debuginfo.
 
-* `packed` - This is the default for Windows MSVC and macOS platforms. The term
+* `packed` - This is the default for Windows MSVC and macOS. The term
   "packed" here means that all the debug information is packed into a separate
   file from the main executable. On Windows MSVC this is a `*.pdb` file, on
-  macOS this is a `*.dSYM` folder, and on other platforms this is a `*.dwp`
-  files.
+  macOS this is a `*.dSYM` folder, and on other platforms this is a collection
+  of `*.dwp` files.
```

Comment:
I couldn't tell when reading this is if it mean one `.dwp` file or a collection of files. I assumed it was the latter, but someone should probably check this.

Reply:
the previous line in the paragraph states "in a single file" so i'm assuming it's just one file here.

Reply:
so, the other changes look great to me, but this one needs to be changed, it seems?